.github/workflows/ac_changelog.yml

```diff
@@ -0,0 +1,39 @@
+name: AC Changelog Lint
+
+on:
+  pull_request:
+    paths:
+      - 'changelogs/fragments/*'
+    branches:
+      - dev
+      - staging*
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      - name: Set up venv
+        run: |
+          python -m pip install --upgrade pip
+          pip install virtualenv
+          mkdir venv
+          virtualenv venv/venv-2.16
```

Comment:
Worth noting that the name for the virtual environment was chosen to trick `ac` into thinking `--venv-setup` was run beforehand